samples/event_manager/src/modules/stats.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+
+#define MODULE stats
+
+#include "measurement_event.h"
+
+#include <logging/log.h>
+#define STATS_LOG_LEVEL 4
+LOG_MODULE_REGISTER(MODULE, STATS_LOG_LEVEL);
+
+#define STATS_ARR_SIZE 10
+
+static s32_t val_arr[STATS_ARR_SIZE];
+
+static void add_to_stats(s32_t value)
+{
+	static size_t ind;
+
+	/* Add new value */
+	val_arr[ind] = value;
+	ind++;
+
+	if (ind >= ARRAY_SIZE(val_arr)) {
```

Comment:
ind cannot be > `ARRAY_SIZE(val_arr)` or you would overflow. Check if equal (==) instead

Reply:
done